deployments/k8s/README.md

```diff
@@ -85,6 +92,11 @@ The [serverless](./serverless) directory contains a set of sample YAML
 resources that you can start from to deploy the agent in a serverless K8s
 environment.
 
+## OpenShift
+
+`configmap.yaml` should have a value at first indent setting `internalHost: ${MY_NODE_IP}`  
+This lets the SignalFx SmartAgent run the `signalfx-agent status` command normally.
```

Comment:
It uses localhost for the status command normally.  Does this mean localhost doesn't work right on OpenShift containers?

Reply:
We saw this at RedHat running their own OpenShift cluster:

```
Could not get status: Get "http://localhost:8095/?section=": dial tcp: lookup localhost on 10.27.167.107:53: no such host
Are you sure the agent is currently running
```

Harnit figured out to do that and it fixed it- I'm not saying its a must, but we should put the suggestion in our docs so that people can find that answer quickly if they have the same problem.